src/layouts/SiteLaunchPad/components/SiteLaunchPadBody.tsx

```diff
@@ -0,0 +1,5 @@
+import { Box, StackProps } from "@chakra-ui/react"
+
+export const SiteLaunchPadBody = ({ children }: StackProps): JSX.Element => {
+  return <Box w="65%">{children}</Box>
```

Comment:
just curious is our CMS UI supposed to be responsive? if so, might need to adjust padding for small screens.

Otherwise, all good

![Screenshot 2023-06-21 at 11 23 30 PM](https://github.com/isomerpages/isomercms-frontend/assets/5507822/3fc0dc66-4942-436a-a12a-90fca518584c)

![Screenshot 2023-06-21 at 11 23 40 PM](https://github.com/isomerpages/isomercms-frontend/assets/5507822/b00fe3bb-5669-49a1-8898-ea00b13723d1)

Reply:
Nope, our CMS is meant to be used on devices with wider aspect ratios!